digital-deck-ui/src/socket.js

```diff
@@ -0,0 +1,3 @@
+import io from 'socket.io-client'
+const socket = io('http://localhost:5000')
```

Comment:
We don't want to just immediately connect the user to the server the moment they start up the app. We should only establish the connection when a user creates/joins a room.

Reply:
I think this is fine for now. This also lets us to access this.socket all over the client side, if it is created in a component then we will have to either create connection for each component or pass the socket somehow that makes the code a little 
messier.

Reply:
Yeah, actually this is fine. I was concerned that there would be multiple socket connections if the the client refreshes the page or something, but apparently socket.io automatically disconnects in that case so there's no problem there.